bophono/data/exceptions-kvp.csv

```diff
@@ -31,6 +44,7 @@
 ས་གདན,sabd-en
 ཁ་གཅོད,khabch-ö
 # dba becomes wa
+དབ,w-a
```

Comment:
`དབ` is `dab`, not `wa`

Reply:
Alright then.
Therefore it seems that replacing:
```
དབ,w-a
དབ/Cb,w-
```
by
```
དབ*/Cb,w-
```
would do the trick.

At least it does for the following tests:
````
    assert_equal_phonetics("KVP", "དབ", "dab")
    assert_equal_phonetics("KVP", "དབོ", "o")
    assert_equal_phonetics("KVP", "དབོས", "ö")
    assert_equal_phonetics("KVP", "དབུ", "u")
    assert_equal_phonetics("KVP", "དབུས", "u")
    assert_equal_phonetics("KVP", "དབི", "i")
    assert_equal_phonetics("KVP", "དབེ", "e")
    assert_equal_phonetics("KVP", "དབང", "ang")
    assert_equal_phonetics("KVP", "དབྱང", "yang")
````

Reply:
well, 

```
assert_equal_phonetics("KVP", "དབོ", "o")
assert_equal_phonetics("KVP", "དབོས", "ö")
assert_equal_phonetics("KVP", "དབང", "ang")
```

should be

```
assert_equal_phonetics("KVP", "དབོ", "wo")
assert_equal_phonetics("KVP", "དབོས", "wö")
assert_equal_phonetics("KVP", "དབང", "wang")
```

but I think instead of having that in exceptions.csv, you can just change the value in roots if necessary:

https://github.com/Esukhia/bophono/blob/master/bophono/data/roots.csv#L139

Reply:
Yep I'm getting tired, obviously wang. Then for the w in wo/wö I also forgot it was mentioned in the issue.

Then, how to define that seems not so simple.

I looked up all words containing dba in all the dictionaries of Steinert's tool, and came up with this list, with proposed phonetics for each case. Once it has been clearly defined what we expect I could move on to the implementation.

```python
assert_equal_phonetics("KVP", "དབ","dab")
assert_equal_phonetics("KVP", "དབག","wak")
assert_equal_phonetics("KVP", "དབགས","wak")
assert_equal_phonetics("KVP", "དབང","wang")
assert_equal_phonetics("KVP", "དབངས","wang")
assert_equal_phonetics("KVP", "དབད","e")
assert_equal_phonetics("KVP", "དབན","en")
assert_equal_phonetics("KVP", "དབནད","en")
assert_equal_phonetics("KVP", "དབའ","wa")
assert_equal_phonetics("KVP", "དབབ","wab")
assert_equal_phonetics("KVP", "དབར","war")
assert_equal_phonetics("KVP", "དབལ","wal")
assert_equal_phonetics("KVP", "དབས","e")
assert_equal_phonetics("KVP", "དབིག","ik")
assert_equal_phonetics("KVP", "དབིབ","i")
assert_equal_phonetics("KVP", "དབུ","u")
assert_equal_phonetics("KVP", "དབུག","uk")
assert_equal_phonetics("KVP", "དབུགས","uk")
assert_equal_phonetics("KVP", "དབུང","ung")
assert_equal_phonetics("KVP", "དབུངས","ung")
assert_equal_phonetics("KVP", "དབུད","ü")
assert_equal_phonetics("KVP", "དབུན","ün")
assert_equal_phonetics("KVP", "དབུབ","ub")
assert_equal_phonetics("KVP", "དབུམ","um")
assert_equal_phonetics("KVP", "དབུའི","ü")
assert_equal_phonetics("KVP", "དབུར","ur")
assert_equal_phonetics("KVP", "དབུརད","ur")
assert_equal_phonetics("KVP", "དབུལ","ül")
assert_equal_phonetics("KVP", "དབུལད","ül")
assert_equal_phonetics("KVP", "དབུས","ü")
assert_equal_phonetics("KVP", "དབེ","e")
assert_equal_phonetics("KVP", "དབེང","eng")
assert_equal_phonetics("KVP", "དབེད","e")
assert_equal_phonetics("KVP", "དབེན","en")
assert_equal_phonetics("KVP", "དབེནད","en")
assert_equal_phonetics("KVP", "དབེམ","em")
assert_equal_phonetics("KVP", "དབེར","er")
assert_equal_phonetics("KVP", "དབོ","wo")
assert_equal_phonetics("KVP", "དབོག","wok")
assert_equal_phonetics("KVP", "དབོགས","wok")
assert_equal_phonetics("KVP", "དབོང","wong")
assert_equal_phonetics("KVP", "དབོན","wön")
assert_equal_phonetics("KVP", "དབོབ","wo")
assert_equal_phonetics("KVP", "དབོའི","wö")
assert_equal_phonetics("KVP", "དབོར","wor")
assert_equal_phonetics("KVP", "དབོལ","wöl")
assert_equal_phonetics("KVP", "དབོལད","wöl")
assert_equal_phonetics("KVP", "དབོས","wö")
assert_equal_phonetics("KVP", "དབྱ","ya")
assert_equal_phonetics("KVP", "དབྱག","yak")
assert_equal_phonetics("KVP", "དབྱང","yang")
assert_equal_phonetics("KVP", "དབྱངས","yang")
assert_equal_phonetics("KVP", "དབྱད","yé")
assert_equal_phonetics("KVP", "དབྱན","yen")
assert_equal_phonetics("KVP", "དབྱར","yar")
assert_equal_phonetics("KVP", "དབྱལ","yal")
assert_equal_phonetics("KVP", "དབྱི","yi")
assert_equal_phonetics("KVP", "དབྱིག","yik")
assert_equal_phonetics("KVP", "དབྱིགས","yik")
assert_equal_phonetics("KVP", "དབྱིང","ying")
assert_equal_phonetics("KVP", "དབྱིངས","ying")
assert_equal_phonetics("KVP", "དབྱིད","yi")
assert_equal_phonetics("KVP", "དབྱིན","yin")
assert_equal_phonetics("KVP", "དབྱིབ","yi")
assert_equal_phonetics("KVP", "དབྱིབས","yi")
assert_equal_phonetics("KVP", "དབྱིལ","yil")
assert_equal_phonetics("KVP", "དབྱིས","yi")
assert_equal_phonetics("KVP", "དབྱུ","yu")
assert_equal_phonetics("KVP", "དབྱུག","yuk")
assert_equal_phonetics("KVP", "དབྱུགས","yuk")
assert_equal_phonetics("KVP", "དབྱུང","yung")
assert_equal_phonetics("KVP", "དབྱུངས","yung")
assert_equal_phonetics("KVP", "དབྱུན","yün")
assert_equal_phonetics("KVP", "དབྱུལ","yül")
assert_equal_phonetics("KVP", "དབྱུས","yü")
assert_equal_phonetics("KVP", "དབྱེ","ye")
assert_equal_phonetics("KVP", "དབྱེན","yen")
assert_equal_phonetics("KVP", "དབྱེནད","yen")
assert_equal_phonetics("KVP", "དབྱེའམ","ye’am")
assert_equal_phonetics("KVP", "དབྱེར","yer")
assert_equal_phonetics("KVP", "དབྱེལ","yel")
assert_equal_phonetics("KVP", "དབྱེས","ye")
assert_equal_phonetics("KVP", "དབྱོད","yö")
assert_equal_phonetics("KVP", "དབྱོལ","yöl")
assert_equal_phonetics("KVP", "དབྲ","dra")
assert_equal_phonetics("KVP", "དབྲག","drak")
assert_equal_phonetics("KVP", "དབྲང","drang")
assert_equal_phonetics("KVP", "དབྲད","dre")
assert_equal_phonetics("KVP", "དབྲབ","drap")
assert_equal_phonetics("KVP", "དབྲལ","dral")
assert_equal_phonetics("KVP", "དབྲལད","dral")
assert_equal_phonetics("KVP", "དབྲས","dre")
assert_equal_phonetics("KVP", "དབྲི","dri")
assert_equal_phonetics("KVP", "དབྲིལ","dril")
assert_equal_phonetics("KVP", "དབྲུལ","drül")
assert_equal_phonetics("KVP", "དབྲེ","dre")
assert_equal_phonetics("KVP", "དབྲེས","dre")
assert_equal_phonetics("KVP", "དབྲོག","drok")
assert_equal_phonetics("KVP", "དབྲོལ","dröl")
assert_equal_phonetics("KVP", "དབླང","lang")
````

Reply:
That looks good to me! Since it's kind of a special case, I'll tell you what I would do but feel free to do otherwise: in roots.csv I would have `x` as the phonetic, and then in the code something like 

`if x and then next vowel in [i, e, u], then '', else w"`

knowing the code it's probably the easiest

Reply:
actually there's probably a more elegant solution, I think having just

```
དབ/Ca,w-
དབོ/Co,w-
```

in exceptions.csv should make everything work without having to change the code (if we have `དབ*,-` in roots.csv). But I'm not totally, the syntax might be a bit different